packages/lib/gpcircuits/scripts/parameters/README.md

```diff
@@ -0,0 +1,3 @@
+# ProtoPODGPC Circuit Parameters
```

Comment:
May not belong in this README in particular, but I suggest writing instructions somewhere about how to generate and test prod artifacts.